Library/Homebrew/formula.rb

```diff
@@ -2281,6 +2281,7 @@ def to_hash
         "url"      => stable_spec.url,
         "tag"      => stable_spec.specs[:tag],
         "revision" => stable_spec.specs[:revision],
+        "using"    => stable_spec.using,
```

Comment:
`using` can be a `Class` and we're probably not going to ever support that.

```suggestion
        "using"    => (stable_spec.using if stable_spec.using.is_a?(Symbol)),
```